apps/api/v2/src/ee/event-types/event-types_2024_06_14/controllers/event-types.controller.ts

```diff
@@ -117,26 +140,26 @@ export class EventTypesController_2024_06_14 {
     if (!eventTypes || eventTypes.length === 0) {
       throw new NotFoundException(`Event types not found`);
     }
-    const eventTypesFormatted = this.eventTypeResponseTransformPipe.transform(eventTypes);
-    const eventTypesWithoutHiddenFields = eventTypesFormatted.map((eventType) => {
+    const eventTypesWithoutHiddenEventTypes = eventTypes.filter((eventType) => !eventType.hidden);
```

Comment:
Also, this filtering could be handled already on repository level by fetching event types with public: false